docs/notebooks/hello_world.ipynb

```diff
@@ -0,0 +1,79 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from edsl import QuestionFreeText"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "You're missing some of the API keys needed to run this job:\n",
+      "     🔑 OPENAI_API_KEY\n",
+      "\n",
+      "You can either add the missing keys to your .env file, or use remote inference.\n",
+      "Remote inference allows you to run jobs on our server.\n",
+      "\n",
+      "🚀 To use remote inference, sign up at the following link:\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace\"><a href=\"https://www.expectedparrot.com/login?edsl_auth_token=Gt4aIsMlSlwayKmzF5vEow\" target=\"_blank\"><span style=\"color: #38bdf8; text-decoration-color: #38bdf8; text-decoration: underline\">https://www.expectedparrot.com/login?edsl_auth_token=Gt4aIsMlSlwayKmzF5vEow</span></a>\n",
```

Comment:
You may wish to make the access token a [repository secret](https://docs.github.com/en/actions/security-for-github-actions/security-guides/using-secrets-in-github-actions#creating-secrets-for-a-repository) rather than putting it in plain text.